resources/fetching/fetcher.go

```diff
@@ -62,25 +62,25 @@ const (
 	GcpServiceUsage   = "gcp-service-usage"
 
 	// Azure resources group subtypes
-	AzureActivityLogAlertType          = "azure-activity-log-alert"
-	AzureBastionType                   = "azure-bastion"
-	AzureClassicStorageAccountType     = "azure-classic-storage-account"
-	AzureDiagnosticSettingsType        = "azure-diagnostic-settings"
-	AzureDiskType                      = "azure-disk"
-	AzureDocumentDBDatabaseAccountType = "azure-document-db-database-account"
-	AzureInsightsComponentType         = "azure-insights-component"
-	AzureMySQLDBType                   = "azure-mysql-server-db"
-	AzureFlexibleMySQLDBType           = "azure-flexible-mysql-server-db"
-	AzureNetworkWatchersFlowLogType    = "azure-network-watchers-flow-log"
-	AzureNetworkWatchersType           = "azure-network-watcher"
-	AzureNetworkSecurityGroupType      = "azure-network-group"
-	AzurePostgreSQLDBType              = "azure-postgresql-server-db"
-	AzureFlexiblePostgreSQLDBType      = "azure-flexible-postgresql-server-db"
-	AzureSQLServerType                 = "azure-sql-server"
-	AzureStorageAccountType            = "azure-storage-account"
-	AzureVMType                        = "azure-vm"
-	AzureVaultType                     = "azure-vault"
-	AzureWebSiteType                   = "azure-web-site"
+	AzureActivityLogAlertType             = "azure-activity-log-alert"
+	AzureBastionType                      = "azure-bastion"
+	AzureClassicStorageAccountType        = "azure-classic-storage-account"
+	AzureDiagnosticSettingsType           = "azure-diagnostic-settings"
+	AzureDiskType                         = "azure-disk"
+	AzureDocumentDBDatabaseAccountType    = "azure-document-db-database-account"
+	AzureInsightsComponentType            = "azure-insights-component"
+	AzureMySQLDBType                      = "azure-mysql-server-db"
+	AzureFlexibleMySQLDBType              = "azure-flexible-mysql-server-db"
+	AzureNetworkWatchersFlowLogType       = "azure-network-watchers-flow-log"
+	AzureNetworkSecurityGroupType         = "azure-network-group"
+	AzurePostgreSQLDBType                 = "azure-postgresql-server-db"
+	AzureFlexiblePostgreSQLDBType         = "azure-flexible-postgresql-server-db"
+	AzureSQLServerType                    = "azure-sql-server"
+	AzureStorageAccountType               = "azure-storage-account"
+	AzureVMType                           = "azure-vm"
+	AzureVaultType                        = "azure-vault"
+	AzureWebSiteType                      = "azure-web-site"
+	AzureBatchedNetworkWatchersByLocation = "azure-batched-network-watchers-by-location"
```

Comment:
nit

```suggestion
	AzureActivityLogAlertType             = "azure-activity-log-alert"
	AzureBastionType                      = "azure-bastion"
	AzureBatchedNetworkWatchersByLocation = "azure-batched-network-watchers-by-location"
	AzureClassicStorageAccountType        = "azure-classic-storage-account"
	AzureDiagnosticSettingsType           = "azure-diagnostic-settings"
	AzureDiskType                         = "azure-disk"
	AzureDocumentDBDatabaseAccountType    = "azure-document-db-database-account"
	AzureFlexibleMySQLDBType              = "azure-flexible-mysql-server-db"
	AzureFlexiblePostgreSQLDBType         = "azure-flexible-postgresql-server-db"
	AzureInsightsComponentType            = "azure-insights-component"
	AzureMySQLDBType                      = "azure-mysql-server-db"
	AzureNetworkSecurityGroupType         = "azure-network-group"
	AzureNetworkWatchersFlowLogType       = "azure-network-watchers-flow-log"
	AzurePostgreSQLDBType                 = "azure-postgresql-server-db"
	AzureSQLServerType                    = "azure-sql-server"
	AzureStorageAccountType               = "azure-storage-account"
	AzureVMType                           = "azure-vm"
	AzureVaultType                        = "azure-vault"
	AzureWebSiteType                      = "azure-web-site"
```

Reply:
I had not realised it was in alphabetical order hehe